metaflow/runtime.py

```diff
@@ -1108,56 +1108,24 @@ def mark_resume_done(self):
 
     def _get_task_id(self, task_id):
         already_existed = True
+        tags = []
```

Comment:
as I mentioned, we just have to check with @darinyu for this but I don't anticipate any issues.